push.go

```diff
@@ -175,15 +177,15 @@ func contains(s []string, e string) bool {
 	return false
 }
 
-func pushPackage() {
+func pushPackage(force *Force) {
```

Comment:
I'm unsure about API design here. We could consider these other function signatures:

```go
func pushPackage(fm *ForceMetadata)
```

or

```go
func (fm *ForceMetadata) pushPackage()
```